db/1.8/modules/ROOT/pages/database-connector-examples.adoc

```diff
@@ -566,6 +566,45 @@ Use DataWeave for all data transformations before the invocation to the callable
 	</db:stored-procedure>
 ----
 
+[[use_pooling_profile_stored_procedures]]
+== Use a Pooling Profile on Stored Procedures
```

Comment:
this title is not meaningful to what this entry explains, this is how to workaround pooling profile limitations when using DB inside of a iterative flow, not how to use a pooling profile on stored procedures

Reply:
I can add this scenario as a new section for the Troubleshooting doc file.
What about this name **Pooling Profiles Connections Making the Flow Hang**

Reply:
I agree to move it to the Troubleshooting file, as well as with the new title.